islandora-support/src/main/java/ca/islandora/alpaca/support/config/PropertyConfig.java

```diff
@@ -77,6 +81,10 @@ public static String addJmsOptions(final String queueString, final int concurren
       builder.append("maxConcurrentConsumers=");
       builder.append(maxConcurrentConsumers);
     }
+    if (asyncConsumers) {
+        builder.append("asyncConsumer=")
```

Comment:
You need a block like
```
if (builder.length() > 0) {
     builder.append('&');
}
```
in here, I'm getting 
```
Caused by: org.apache.camel.ResolveEndpointFailedException: Failed to resolve endpoint: broker://queue:islandora-connector-ocr?concurrentConsumers=1asyncConsumer=true due to: Error binding property (concurrentConsumers=1asyncConsumer=true) with name: concurrentConsumers on bean: broker://queue:islandora-connector-ocr?concurrentConsumers=1asyncConsumer=true with value: 1asyncConsumer=true
```
when I set some of the options.

Reply:
Should be before the line I referenced...but I know you know that. 😉

Reply:
Derp... had it locally, forgot to push it.